karma.conf.babel.js

```diff
@@ -0,0 +1,80 @@
+import paths from './paths';
```

Comment:
Karma [doesn't support](https://github.com/karma-runner/karma/issues/1597) a babel register hook as gulp and webpack do, yet.  We write our karma babel file in future ES.  Then, karma runs `karma.conf.js` which uses the [babel register hook](https://babeljs.io/docs/usage/require/) to compile this file at run time.